content/start.html

```diff
@@ -1,53 +1,45 @@
 <h2>Welcome</h2>
 
-<p> What you will find here is a lot of plugin stuff for <a href=
-"http://www.geany.org">Geany</a>. There is a Geany Plugins project</a>
-containing a lot of plugins, which is developed by various developers on <a
-href="https://github.com/geany/geany-plugins">GitHub</a>. </p>
+<p>This site is the home page for <a href= "http://www.geany.org">Geany</a>
+plugins. It includes official Geany Plugins, as well as 3rd party ones.</p>
 
-<p> Any plugin delivered with the Geany Plugins project is listed on the
-left sidebar (Or rather any plugin providing a valid README file written in
-reStructuredText. There are some plugins in an early development state, not
-having such a README file yet.) </p>
+<p>Some plugins may lack details. Feel free to bug us
```

Comment:
actually the authors need to provide this or the community need to add this. @dmaphy and me are mainly just accepting PR and so on. I'm not sure if someone would actually take this bug.

Reply:
Exactly. Lack of information means there is no README file and this has to be reported to the plugin maintainer. not the website maintainers for plugins.geany.org. :-)